ui/packages/statement/src/components/StatementsOverview.tsx

```diff
@@ -343,9 +344,15 @@ export default function StatementsOverview({
           onChange={handleTimeRangeChange}
         >
           {state.timeRanges.map(item => (
-            <Option value={item.begin_time} key={item.begin_time}>
-              {item.begin_time} ~ {item.end_time}
-            </Option>
+             () => {
+                if (item.begin_time) {
```

Comment:
I don't think it it is a good idea to use `if...else` inside the jsx, and why not just use `item.begin_time && <Option ...>`.

BTW, here I don't think we need to judge `item.begin_time`, because if `item.begin_time` has no value, the `state.timeRanges` will be `[]`, the `.map()` won't be called.

Reply:
I'll try tonight, I remember the code can not compile if not change.